### PR TITLE
app boilerplate done, building out app structure

### PR DESCRIPTION
**Pull request by RichardSPrins**

### CODE REVIEW

File: redux-todo/src/components/Todo.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+const Todo = props =>  {
```
Comment (Zealll) [inline redux-todo/src/components/Todo.js]: What is the purpose of this Component? Is it doing anything?

File: redux-todo/src/components/TodoForm.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          <input 
+            type="text" 
+            name="todoText" 
+            value={this.state.todoText} 
+            onChange={this.handleInputChange}
+            required
+          />
+          <button>Add</button>
+        </form>
+        {this.props.todoStateFromProps.map(todo => <p style={todo.completed ? completedStyle : null} onClick={() => this.toggleTask(todo.id) }>{todo.value}</p>)}
```
Comment (Zealll) [inline redux-todo/src/components/TodoForm.js]: Try to not use inline styling, it's better to use className, and manage all the styling in your CSS. You can also do a Ternary operator on a className

File: redux-todo/src/components/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import Todo from './Todo'
+
+ class TodoList extends Component {
```
Comment (Zealll) [inline redux-todo/src/components/TodoList.js]: What about this Component?

File: redux-todo/src/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```
Comment (Zealll) [inline redux-todo/src/index.js]: Try to delete serviceWorker since we don't use it

File: redux-todo/src/index.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+ReactDOM.render(
+  <Provider store={store}>
+  <App />
+  </Provider>, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (Zealll) [inline redux-todo/src/index.js]: This one as well

File: redux-todo/src/serviceWorker.js [hunk 6]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```
Comment (Zealll) [inline redux-todo/src/serviceWorker.js]: Delete the service worker JS file as well

**Comment by RichardSPrins** [comment]
Remember we put all of my component code I to one component and the rest
became useless, therefore I'm not importing them

**Pull request merged**